saleor/data_feeds/google_merchant.py

```diff
@@ -123,8 +125,12 @@ def item_tax(item: ProductVariant, discounts: Iterable[DiscountInfo]):
     Read more:
     https://support.google.com/merchants/answer/6324454
     """
-    # FIXME https://github.com/mirumee/saleor/issues/4311
-    return "US::%s:y" % zero_money()
+    country = Country(settings.DEFAULT_COUNTRY)
+    tax_rate = get_plugins_manager().get_tax_rate_percentage_value(
+        item.product.product_type, country
+    )
+    tax_ship = "yes" if charge_taxes_on_shipping() else "no"
```

Comment:
Since `charge_taxes_on_shipping` doesn't depend on the current `item`, maybe this could be executed outside of `item_tax` function e.g. at the top of the module? Now it would be called for each item in the feed and internally it makes a database query. We could have only one query for the entire feed.

Reply:
perfect, working on it. Thanks

Reply:
@maarcingebala I applied the changes, please take a look again. Btw, The Codecov report it is failing, I think it's related to the `utils.py` file in `saleor/order/` but I didn't add any change there. Please let me know if there's a way to fix it.

Reply:
Hi @maarcingebala I think I fixed the issue mentioned before. Please take a look again.

Thanks,

Reply:
Looks really good 👍